frontend/components/CellOutput.js

```diff
@@ -333,8 +333,19 @@ export let RawHTMLContainer = ({ body, persist_js_state = false, last_run_timest
                 for (let code_element of container.current.querySelectorAll("code")) {
                     for (let className of code_element.classList) {
                         if (className.startsWith("language-")) {
+                            let aliases = {
+                                "html": "htmlmixed",
+                                "jl": "julia",
+                                "js": "javascript",
+                            }
```

Comment:
More aliases could be added here (e.g., `jldoctest`).

(Another thing: maybe this variable should be 'const` and outside of the local scope?)

Reply:
Style-wise, declaring it here seems like the best option.

Performance-wise, that seems like over-optimization, and the JS compilers might actually be clever enough to do it automatically.

Reply:
Great, I'll leave it like that.